# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/sparc64-asm-page.h.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

During strace compilation page.h is included and we do not needlessly want to bail it out.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.12/include/asm-sparc64/page.h.orig	2006-08-23 16:29:58.000000000 +0000
+++ linux-2.6.12/include/asm-sparc64/page.h	2006-08-23 16:30:14.000000000 +0000
@@ -15,7 +15,7 @@
 #elif defined(CONFIG_SPARC64_PAGE_SIZE_4MB)
 #define PAGE_SHIFT   22
 #else
-#error No page size specified in kernel configuration
+#warning No page size specified in kernel configuration
 #endif
 
 #define PAGE_SIZE    (_AC(1,UL) << PAGE_SHIFT)
